src/controllers/github.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express';
+import BaseController from './base.controller';
+import GithubService from '../services/github.service';
+
+class GithubController extends BaseController {
+	service: GithubService;
+
+	public constructor() {
+		super();
+		this.service = new GithubService();
+	}
+
+	getContributors = async (_: Request, res: Response) => {
+		try {
+			const contributors = await this.service.getContributors(
+				'evrimagaci',
```

Comment:
projects names are; 
 - `periodum`
 - `periodum-api`
 - `periodum-apple`

Maybe we can also provide these names as an array (can be split by `,`)  in `env` _variables`?

```bash
GITHUB_REPOS="periodum,periodum-api,periodum-apple"
```

In the future we might add new repos. So, we won't need to re-deploy the app but changing the env variable.